src/render.js

```diff
@@ -21,7 +21,7 @@ export function render(vnode, parentDom, replaceNode) {
 	diffChildren(
 		parentDom,
 		replaceNode ? vnode : (parentDom._prevVNode = vnode),
-		replaceNode ? undefined : oldVNode,
```

Comment:
what was the point of having undefined here?

Reply:
Hmm I don't really know @LukasBombach was there a reason to put this on undefined back then? Maybe I'm breaking something that isn't tested?

Reply:
I think I got it wrong there. I thought there was no oldVNode if your hydrate from nothing, but it was probably just a brainfart